src/redux/features/votes/votesSlice.ts

```diff
@@ -0,0 +1,46 @@
+import { ActionReducerMapBuilder, PayloadAction, createSlice } from '@reduxjs/toolkit';
+
+import { StatusT, buildReducers } from 'src/redux/features/asyncUtils';
+import { requestProjectVotesDelete, requestProjectVotesUpdate } from 'src/redux/features/votes/votesAsyncThunks';
+import { GetProjectVotesResponse, GetProjectVotesResponsePayload } from 'src/types/Votes';
+
+// Define a type for the slice state
+type Data = {
+  error?: string;
+  votes?: GetProjectVotesResponsePayload;
+};
```

Comment:
Can't find my earlier comment.
Suggest making this a record of project id to votes response.
example:
```
// Define a type for the slice state
type Data = Record<number, StatusT<GetProjectVotesResponsePayload>>;

// Define the initial state
const initialState: Data = {};

export const votesSlice = createSlice({
  name: 'votesSlice',
  initialState,
  reducers: {},
  extraReducers: (builder: ActionReducerMapBuilder<Data>) => {
    buildReducers(requestProjectVotes, true)(builder);
  },
});

```

Reply:
I was wondering about keying off of the projectId when I set it up. I peaked in a few slices and it didn't seem like we're currently doing that in any of the other reducers, at least for projectId.